frontend/src/ui-components/AcmTable/AcmTable.test.tsx

```diff
@@ -899,22 +899,8 @@ describe('AcmTable', () => {
         />
       </MemoryRouter>
     )
-    userEvent.click(getByTestId('expandable-toggle0'))
+    userEvent.click(getByTestId('expand-toggle0'))
     expect(getByTestId('expanded')).toBeInTheDocument()
-
-    // Run delete action for code coverage (no delete support on expanded content)
-    userEvent.click(getAllByLabelText('Actions')[1])
-    await waitFor(() => expect(getByRole('menu')).toBeVisible())
-    expect(getByText('Delete item')).toBeVisible()
-    userEvent.click(getByText('Delete item'))
-    expect(expandedDeleteAction).not.toHaveBeenCalled()
-
-    // Run tooltipped delete action for code coverage (no delete support on expanded content)
-    userEvent.click(getAllByLabelText('Actions')[1])
-    await waitFor(() => expect(getByRole('menu')).toBeVisible())
-    expect(getByText('Delete item tooltip')).toBeVisible()
-    userEvent.click(getByText('Delete item tooltip'))
-    expect(expandedDeleteAction).not.toHaveBeenCalled()
```

Comment:
Unnecessary tests. They don't cover real scenarios and were being used for increase coverage.